### PR TITLE
Remaining issues to be wrapped up in preparation for deployment to production

### PR DESCRIPTION
**Pull request by lmcmicu**

### CODE REVIEW

File: src/clovertonemusic/html.clj [hunk 1]
```diff
@@ -1641,54 +1641,54 @@
...
-                                    [:a.download
-                                     {:href (str "/purchases/" purchaseid "/"
-                                                 (get-filename chart) ".score.pdf")} "Score"]
-                                    [:a.download
-                                     {:href (str "/purchases/" purchaseid "/"
-                                                 (get-filename chart) ".parts.pdf")} "Parts"]]])
-                                [:tr [:td] [:td] [:td] [:td [:hr]]]
-                                [:tr [:th] [:th] [:th "Subtotal"]
-                                 [:td {:data-label "Subtotal"} (format "$%.2f" subtotal)]]
+    (when (and purchase-summary purchase-details)
```
Comment (lmcmicu) [inline src/clovertonemusic/html.clj:1785]: I would suggest clicking the "hide whitespace" option when viewing this part of the diff.

File: src/clovertonemusic/data.clj [hunk 2]
```diff
@@ -406,17 +406,41 @@
                    (apply distinct?))
       (fail "User database contains duplicate emails"))))
 
+(defn- filter-out-empty-purchases
```
Comment (lmcmicu) [inline src/clovertonemusic/data.clj]: See James's comment: https://github.com/jamesaoverton/clovertonemusic/issues/20#issuecomment-674229490

**Comment by lmcmicu** [comment]
Changes to the `config.edn` need to be made as a result of https://github.com/jamesaoverton/clovertonemusic/pull/21/commits/937e2de2e1914343b05d2295edbe442bc62f413a and https://github.com/jamesaoverton/clovertonemusic/pull/21/commits/8a252d8cbe16cf22c8623c4289583cb975be70f0:

* operating environment and log-levels are now given as keywords, not as strings
```
{:env :dev

  ;; Should be one of :debug, :info, :warn, :error, :fatal
 :log-level {:dev :info
             :test :info
             :prod :info}
```

* A new parameter for the "operations team" email list has been added:

```
 ;; List of email addresses that should be notified of significant events:
 :admin-ops-email-list {
    :dev ["email-1" "email-2" "email-3"]
    :test ["email-1" "email-2" "email-3"]
    :prod ["email-1" "email-2" "email-3"]}

```

File: project.clj [hunk 3]
```diff
@@ -21,5 +21,7 @@
                  [ring/ring-core "1.8.1"]
                  [ring/ring-json "0.5.0"]]
   :main clovertonemusic.core
+  :jar-exclusions [#"^public\/"]
+  :uberjar-exclusions [#"^public\/"]
```
Comment (jamesaoverton) [inline project.clj]: Running from the uberjar, the `/application.css` is not served, which breaks the whole site. I'm going to remove these lines.

**Pull request merged**